src/Expression/AbstractArrayExpressionBuilder.php

```diff
@@ -68,13 +69,17 @@ public function __construct(protected readonly QueryBuilderInterface $queryBuild
     /**
      * The Method builds the raw SQL from the `$expression` that won't be additionally escaped or quoted.
      *
-     * @param ArrayExpression $expression The expression to build.
+     * @param ExpressionInterface $expression The expression to build.
```

Comment:
We can leave `ArrayExpression` here since it is the only valid type for the class method.